src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -1,211 +1,536 @@
+                expectedNoteErrorMessage);
+
+        String expectedProjectErrorMessage =
+                EditProjectCommand.MESSAGE_NOT_EDITED;
+
+        // missing all fields in project prefix
+        assertParseFailure(parser, VALID_ITEM_INDEX + ITEM_TYPE_PROJECT,
+                expectedProjectErrorMessage);
+
+        String expectedResumeErrorMessage =
+                EditResumeCommand.MESSAGE_NOT_EDITED;
+
+        // missing all fields in resume prefix
+        assertParseFailure(parser, VALID_ITEM_INDEX + ITEM_TYPE_RESUME,
+                expectedResumeErrorMessage);
+
+        String expectedSkillErrorMessage =
+                EditSkillCommand.MESSAGE_NOT_EDITED;
+
+        // missing all fields in skill prefix
+        assertParseFailure(parser, VALID_ITEM_INDEX + ITEM_TYPE_SKILL,
+                expectedSkillErrorMessage);
+    }
+
+    @Test
+    public void parse_invalidValue_failure() {
+        // invalid name
+        assertParseFailure(parser,
+                VALID_ITEM_INDEX + ITEM_TYPE_INTERNSHIP + INVALID_NAME_DESC
+                        + PREFIXED_ROLE_FRONTEND + PREFIXED_TIME_FROM
+                        + PREFIXED_TIME_TO + PREFIXED_INTERNSHIP_DESCRIPTION + PREFIXED_TAG_FRONTEND,
+                Name.MESSAGE_CONSTRAINTS);
+
+        // invalid from
+        assertParseFailure(parser,
+                VALID_ITEM_INDEX + ITEM_TYPE_INTERNSHIP + PREFIXED_NAME_GOOGLE
+                        + PREFIXED_ROLE_FRONTEND + INVALID_FROM_DESC
+                        + PREFIXED_TIME_TO + PREFIXED_INTERNSHIP_DESCRIPTION + PREFIXED_TAG_FRONTEND,
+                Time.MESSAGE_CONSTRAINTS);
+
+        // invalid to
+        assertParseFailure(parser,
+                VALID_ITEM_INDEX + ITEM_TYPE_INTERNSHIP + PREFIXED_NAME_GOOGLE
+                        + PREFIXED_ROLE_FRONTEND + PREFIXED_TIME_FROM
+                        + INVALID_TO_DESC + PREFIXED_INTERNSHIP_DESCRIPTION + PREFIXED_TAG_FRONTEND,
+                Time.MESSAGE_CONSTRAINTS);
+
+        // invalid tag
+        assertParseFailure(parser,
+                VALID_ITEM_INDEX + ITEM_TYPE_INTERNSHIP + PREFIXED_NAME_GOOGLE
+                        + PREFIXED_ROLE_FRONTEND + PREFIXED_TIME_FROM
+                        + PREFIXED_TIME_TO + PREFIXED_INTERNSHIP_DESCRIPTION + INVALID_TAG_DESC,
+                Tag.MESSAGE_CONSTRAINTS);
+
+        // invalid website
+        assertParseFailure(parser,
+                VALID_ITEM_INDEX + ITEM_TYPE_PROJECT + PREFIXED_NAME_ORBITAL + PREFIXED_TIME_TO
+                        + INVALID_WEBSITE_DESC + PREFIXED_DESCRIPTION_ORBITAL,
+                Website.MESSAGE_CONSTRAINTS);
+
+        // invalid level
+        assertParseFailure(parser,
+                VALID_ITEM_INDEX + ITEM_TYPE_SKILL + PREFIXED_NAME_REACT
+                        + INVALID_LEVEL_DESC + PREFIXED_TAG_FRONTEND,
+                "A skill level can only be basic, intermediate, or advanced.");
```

Comment:
I think I also changed this somewhere

Reply:
I can't remember where I made the change, but you can make use of the class-level constant of `Level` [here](https://github.com/AY1920S2-CS2103T-F10-1/main/blob/887256e2dfb825569b50ada766b38c81166d3bdb/src/main/java/seedu/address/model/item/field/Level.java#L11).

You can use `Level.MESSAGE_CONSTRAINTS` instead

Reply:
Yes it's in Level :D